system_tests/system_tests.go

```diff
@@ -415,6 +407,22 @@ CONSOLE_LOG=new`
 					Expect(err).NotTo(HaveOccurred())
 					publishAndConsumeSTOMPMsg(hostname, rabbitmqNodePort(ctx, clientSet, cluster, "stomps"), username, password, cfg)
 				})
+
+				By("disabling non TLS listeners", func() {
+					// verify that rabbitmq.conf contains listeners.tcp = none
+					cfgMap := getConfigFileFromPod(namespace, cluster, "/etc/rabbitmq/rabbitmq.conf")
+					Expect(cfgMap).To(HaveKeyWithValue("listeners.tcp", "none"))
```

Comment:
Let's add two more expectations here because that system test already enables MQTT, STOMP, TLS, and the flag `DisableNonTLSListeners`?
```
Expect(cfgMap).To(HaveKeyWithValue("mqtt.listeners.tcp", "none"))
Expect(cfgMap).To(HaveKeyWithValue("stomp.listeners.tcp", "none"))
```

Reply:
Good point David. Updated and will merge now 👍